README.md

```diff
@@ -19,7 +19,7 @@ Features include:
 * Showing figures inline in Jupyter notebooks, Juno and VSCode-julia. Both png- and svg-figures can be shown.
 * Exporting to tex, pdf, svg, and png, file formats.
 * Customizing the preamble so that commands from latex packages can be used.
-* [Extra functionality](https://kristofferc.github.io/PGFPlotsX.jl/stable/examples/juliatypes/) when different packages are loaded, for example *Colors*, *DataFrames*, *Contour* etc.
+* [Extra functionality](https://kristofferc.github.io/PGFPlotsX.jl/stable/examples/juliatypes.html) when different packages are loaded, for example *Colors*, *DataFrames*, *Contour* etc.
```

Comment:
Thanks for noticing this. Instead of adding `.html`, I would suggest just removing the trailing `/`.

Reply:
Ok, changed the pull request now to follow your suggestion.